expyfun/stimuli/_tracker.py

```diff
@@ -239,6 +248,22 @@ def respond(self, correct):
                 if self._direction <= 0:
                     self._direction = 1
 
+        if self._repeat_limit == 'reversals':
+            if self._x[-1] == self._x_min:
```

Comment:
> Maybe this gets more complicated when it's not 1-up, 1-down, I'm not sure...

It almost certainly would... 

Hitting the limits more than just a quick brush is going to hose any threshold estimate you might do, so maybe it isn't worth all this thought.

@maddycapp27 Please do test it with a 2-down 1-up, or even a 3-down, 2-up track to make sure it doesn't do anything crazy.